src/main/java/com/laserfiche/repository/api/RepositoryApiClientImpl.java

```diff
@@ -56,6 +56,8 @@ public static RepositoryApiClient CreateFromAccessKey(String servicePrincipalKey
     @Override
     public void setDefaultRequestHeaders(Map<String, String> defaultRequestHeaders) {
         defaultHeaders = defaultRequestHeaders;
+        String defaultHeadersKey = defaultHeaders.keySet().toArray()[0].toString();
+        httpClient.config().addDefaultHeader(defaultHeadersKey, defaultHeaders.get(defaultHeadersKey));
```

Comment:
if this function is called multiple times, will all headers be added as default or will it reset the previous default headers? For example if I call this function multiple times like this
`setDefaultRequestHeaders({ header1: value, header2: value })`
`setDefaultRequestHeaders({ header3: value })`
i guess i would expect only `header3` is sent as a default header since we dont have a way to reset the headers and this is how the other client libraries work.

Reply:
after changing to function call to `setDefaultHeader`, you should only expect `header3` to be the default header